### PR TITLE
Update instructions for newbie users like me

### PR DESCRIPTION
**Pull request by zenorocha**
Hey @benknoble @dsifford,

I never used Vim and today I tried to install the theme. My goal was to use Dracula on MacVim and also on my terminal. Since I'm not used to it, I must admit I had some trouble to configure it. This issue https://github.com/dracula/vim/issues/157 is what helped me to enable it. That's why I'm sending this PR - it's an attempt to facilitate the instructions for other people.

Once again, I'm not an expert, so feel free to request changes.

> Note: In order to make it work on my terminal, I also had to add these files to my `~/.vimrc` file. I'm not sure if we should add them in the instructions too or if I missed something.

```
set t_Co=256
set term=xterm
set background=dark
```

### Machine Info

- **Vim type** vim and MacVim
- **Vim version**: 8.1.1312
- **OS**: Darwin 19.3.0 Darwin Kernel Version 19.3.0: Thu Jan  9 20:58:23 PST 2020; root:xnu-6153.81.5~1/RELEASE_X86_64 x86_64
- **Terminal/Terminal Emulator/VTE**: iTerm, Hyper and Mac Terminal
- **`TERM` environment variable**: xterm-256color

### CODE REVIEW

**Comment by dsifford** [comment]
LGTM. Let's let @benknoble give this a quick once-over before merging though since he uses vim's default plugin system. :+1:

File: INSTALL.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
 ### [Vim](http://www.vim.org/)
 
-#### Install using Git
+#### Install
+
+These are the default instructions using Vim 8's `|packages|` feature. See sections below, if you use other plugin managers.
+
+1. Create theme folder (in case you don't have yet):
+
+	```
```
[the remark this answers is not in the document]
Comment (benknoble) [reply INSTALL.md]: Also, the `themes` part of that path is technically arbitrary

File: INSTALL.md [hunk 2]
```diff
@@ -1,6 +1,42 @@
 ### [Vim](http://www.vim.org/)
 
-#### Install using Git
+#### Install
+
+These are the default instructions using Vim 8's `|packages|` feature. See sections below, if you use other plugin managers.
```
[the remark this answers is not in the document]
Comment (benknoble) [reply INSTALL.md]: Considering I don't think we've done this consistently, I'm going to say don't worry about this; I'll do it myself another time if I think it matters.

**Comment by zenorocha** [comment]
I'm ok with that.

**Pull request merged**